### DISCUSSION TITLE
Quic support in .net

### DISCUSSION THREAD

**Discussion by kamranattari4192** [microsoft/msquic#3659] [Q&A] [answered]
Hi,

Currently, Quic support in .net 7.0 is in preview as mentioned here https://learn.microsoft.com/en-us/dotnet/fundamentals/networking/quic/quic-overview. Can I know when it will be fully supported in .NET?

**Comment by ManickaP** [comment, marked as answer]
We're working on making that happen in .NET 8.0, but as we're swamped with other issues this release, it's still not 100% it'll happen. If you have any feedback on the API shape, feel free to let us know in https://github.com/dotnet/runtime.
